docs/changelog.md

```diff
@@ -7,6 +7,14 @@ follow [merged Pull request
 pages](https://github.com/dalibo/ldap2pg/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged).
 
 
+# Unreleased
+
+- Don't manage public by default. This allow to let ldap2pg manage a subset of
```

Comment:
I'm confused. This says "Don't manage public by default", but I see you've added `'public'` to `defaults.py`.

Also `inspector.py` has a comment: "Legacy ldap2pg manages public, always. We keep this by default"

Reply:
You're right. It's a non-sense. Thanks !

Reply:
Updated. Is it clearer ?